src/lib/ecr_repos.py

```diff
@@ -26,11 +26,23 @@ def all_repos(self):
       [repo.name for repo in self._repos]
     )
 
-  def get_repo(self, repo_name):
-    ''' return ECRRepo object whose name matches repo_name '''
+  def repos_with_name(self, str1, exact_match=False):
```

Comment:
`str1` variable would benefit from a more descriptive name

Reply:
sure, any suggestions?

Reply:
maybe `string_to_match`?

Reply:
works for me.